<launch>
    <arg name="image_topic" default="/lbr/storz_endoscope_camera/image_raw"/>

    <group ns="lbr">
        <!-- load params -->
        <rosparam command="load" file="$(find rcom_impl)/config/four_dof_params.yml"/>

        <!-- launch action server 
                - runs under action_server: 'RCoM_ActionServer', defined in config/params.yml @ rcom_impl package
        -->
        <node name="four_dof_rcom_node" pkg="rcom_impl" type="four_dof_rcom_node" output="screen"/>

        <!-- launch action client node
                - subscribes to camera/image_raw
                - publishes to visual_servo/img0
        -->
        <node name="rcom_init_node" pkg="h_rcom_vs" type="rcom_init_node" output="screen">
            <remap from="camera/image_raw" to="$(arg image_topic)"/>
        </node>

    </group>

</launch>